src/mockups/ATokenMock.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.8.10;
+
+// interfaces
+import "../../lib/solmate/src/tokens/ERC20.sol";
+import "../interfaces/IAToken.sol";
+
+contract ATokenMock is ERC20 {
+    address public uToken;
```

Comment:
Not a problem but more out of curiosity :p, why uToken and not aToken?

Reply:
underlying

Reply:
in erc4626 its called "asset" and in aave's impl its called UNDERLYING_ASSET_ADDRESS.

Reply:
Makes sense